src/metpy/io/gempak.py

```diff
@@ -2677,6 +2678,10 @@ def nearest_time(self, date_time, station_id=None, station_number=None):
         station_number : int or Sequence[int]
             Station number of the surface station.
 
+        parse_special : bool
```

Comment:
Would `include_special` be a better name (here and elsewhere)? `parse_special` just struck me a little odd for a function called `nearest_time`.